suites/pacific/rados/deploy-stretch-cluster-mode.yaml

```diff
@@ -155,7 +155,7 @@ tests:
   - test:
       name: Deploy stretch Cluster
       module: test_deploy_stretch_cluster_baremetal.py
-      polarion-id: CEPH-83573621, CEPH-83572692
+      polarion-id: CEPH-83573625
```

Comment:
@pdhiran 
CEPH-83573621: Polarion ID is part of other test suites so it is okay to skip here
CEPH-83572692: This polarion ID is part of only this test case, we should preserve it here
CEPH-83573625: we can include this along side CEPH-83572692
And do we need to make these changes for Quincy as well?

@SrinivasaBharath Please include the final changes in Reef

Reply:
Sure, I'll make similar changes for Pacific and Quincy. Sounds good!